x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/string/InsertFunctionProcessor.java

```diff
@@ -48,21 +48,15 @@ public Object process(Object input) {
     }
 
     public static Object doProcess(Object input, Object start, Object length, Object replacement) {
-        if (input == null) {
+        if (input == null || start == null || length == null || replacement == null) {
             return null;
         }
         if ((input instanceof String || input instanceof Character) == false) {
             throw new SqlIllegalArgumentException("A string/char is required; received [{}]", input);
         }
-        if (replacement == null) {
-            return input;
-        }
-        if ((replacement instanceof String || replacement instanceof Character) == false) {
+        if (!(replacement instanceof String || replacement instanceof Character)) {
```

Comment:
Nit: was the change intentional (here and one more spot in the patch)?
(Myself I dislike `== false`, but since this the direction [the train is going](https://github.com/elastic/elasticsearch/blob/master/CONTRIBUTING.md#java-language-formatting-guidelines), I'm comfier with a consistent style.)

Reply:
Thanks for pointing out. No, it wasn't, I think just IDEA tried to be helpful (will revert this one).

Reply:
Reverted.